src/tv2-common/migrations/index.ts

```diff
@@ -173,6 +174,74 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string, fromValue: string, targetTable: string) {
```

Comment:
Not really a need to postfix `Str` in `version`

Reply:
It seems to be there because `version` is the property name and the linter doesn't allow that.